worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -870,6 +870,45 @@ namespace RTC
 
 		if (isSyncPacket)
 		{
+			if (!lastSentRtpPacketHasMarker && shouldSwitchCurrentSpatialLayer)
+			{
+				RTC::RtpPacket::Header header;
+				header.csrcCount      = 0u;
+				header.extension      = 0u;
+				header.padding        = 0u;
+				header.version        = 2u;
+				header.payloadType    = packet->GetPayloadType();
+				header.marker         = 1u;
+				header.sequenceNumber = seq;
+				header.timestamp      = this->rtpStream->GetMaxPacketTs();
+				header.ssrc           = packet->GetSsrc();
+
+				RTC::RtpPacket* emptyPacket =
+				  RTC::RtpPacket::Parse(reinterpret_cast<uint8_t*>(&header), sizeof(RTC::RtpPacket::Header));
```

Comment:
Note for us: we should provide a proper `RtpPacket` factory function. This code sucks (but this is the only that is possible for now).

To clarify, nothing to be done within this PR. This is a task we should deal with in the future.

Reply:
Said that the issus can still be reproduced after applying the patch.

We have two chrome browsers of the same version as video source. And another chrome broswer to watch them.
One of the video source had mosics when switching layers, but the other did't.